readthedocs/rtd_tests/tests/projects/test_version_sorting.py

```diff
@@ -65,3 +71,95 @@ def test_sort_alpha(self):
             ['latest', 'carrot', 'banana', 'apple'],
             [v.slug for v in sort_version_aware(versions)],
         )
+
+    def test_sort_git_master(self):
+        identifiers = ['master', '1.0', '2.0', '1.1', '1.9', '1.10']
```

Comment:
Could use a test that sorts both `master` and `latest` to confirm their ordering relative to each other as well. Not super important tho.l

Reply:
Done